src/poprox_storage/repositories/newsletters.py

```diff
@@ -56,6 +57,41 @@ def store_newsletter(self, newsletter: Newsletter):
                 )
                 self.conn.execute(stmt)
 
+    def store_newsletter_feedback(self, account_id: list[UUID], newsletter_id: list[UUID], ispositive: bool):
```

Comment:
- why lists?
- And should `ispositive` be `is_positive` for code style?
- And should it be bool|none to allow updating back to null?

Reply:
Code below is correct, but the type annotations need changes. Agreed that `is_positive` would be better.